enigma-hs.cabal

```diff
@@ -19,16 +19,17 @@ library
   ghc-options:         -Wall
   exposed-modules:     Crypto.Enigma, Crypto.Enigma.Helper
   exposed:             True
-  -- other-modules:       
-  -- other-extensions:    
-  build-depends:       base >=4.6
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base >=4.6,
+                       QuickCheck
```

Comment:
Ideally only QuickCheck should be in the test-suite, but I think the cabal file is a bit screwed up, e.g. the executable is in the tests folder. You're welcome to fix it, but that's not necessary for the issue at hand.